ci/build_container/build_recipes/benchmark.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -x
+
+export COMMIT="e1c3a83b8197cf02e794f61228461c27d4e78cfb"  # benchmark @ Jan 11, 2018
+
+git clone https://github.com/google/benchmark.git
+(cd benchmark; git reset --hard "$COMMIT")
+mkdir build
+
+cd build
+cmake -G "Unix Makefiles" ../benchmark \
+  -DCMAKE_BUILD_TYPE=RELEASE \
+  -DBENCHMARK_ENABLE_GTEST_TESTS=OFF
+make
+cp src/libbenchmark.a "$THIRDPARTY_BUILD"/lib
+cd ../benchmark
+
+pwd
+include_dir="$THIRDPARTY_BUILD/include/testing/base/public"
```

Comment:
Prefer `INCLUDE_DIR` for consistency with other variables (I know in Google we use lowercase, but we would need to switch everything over).

Reply:
done; FWIW I thought lower-case variables were common in bash, and upper-case variables when they are exported.

Reply:
They are, and it's in the Google style guide, but we haven't had that convention in Envoy, so we tend to opt to be consistent with the existing code base as the lesser evil.